core/flow/tc_throttling_test.go

```diff
@@ -100,3 +100,35 @@ func TestThrottlingChecker_DoCheckQueueingParallel(t *testing.T) {
 	// Non-strict mode may not be strictly accurate, so here we tolerate a delta.
 	assert.InEpsilon(t, threshold/(float64(intervalMs)/1000.0), waitCount, 1)
 }
+
+func TestThrottlingChecker_DoCheckParallelPass(t *testing.T) {
+	intervalMs := 10000
+	threshold := 50.0
+	timeoutMs := 0
+
+	tc := NewThrottlingChecker(nil, uint32(timeoutMs), uint32(intervalMs))
+
+	wg := &sync.WaitGroup{}
+	beginWg := &sync.WaitGroup{}
+	gc := 100000
```

Comment:
A large number of goroutine are responsible for the CI error.
```
=== RUN   TestThrottlingChecker_DoCheckParallelPass
race: limit on 8128 simultaneously alive goroutines is exceeded, dying
```

Reply:
I changed it to 512.